### ISSUE TITLE
Need minAttributes and maxAttributes in the swagger:model annotation

### ISSUE THREAD

**Issue by seanhoughton** [go-swagger/go-swagger#2985]
## Problem statement

The spec-to-code transformation supports the `minProperties` and `maxProperties` constraints but the code-to-spec transformation does not. This means that users who use both transformations cannot take advantage of [the fix to untyped maps](https://github.com/go-swagger/go-swagger/pull/2477) without manually adding the constraints in the generated swagger spec.

## Model specification

```
// swagger:model MyObjectType
// minAttributes: 0  <--- would like to see support for this
type MyObjectType map[string]interface{}
```

I spent some time in the code trying to figure out how to do this, but it would be more productive if someone can give me some pointers - I'm happy to work on a PR.

Great project!

**Comment by fredbi** [comment]
Yes good proposal. minProperties, maxProperties are more in line with jsonschema validations, though
